### PR TITLE
[RAPPS-DB] Add Core Temp 1.18.1

### PR DESCRIPTION
**Pull request by scropool**
image
working

### CODE REVIEW

**Review by vsaucing — APPROVED** [review]

**Review by learn-more — APPROVED** [review]

**Review by julenuri — CHANGES_REQUESTED** [review]

**Review by julenuri — APPROVED** [review]

**Comment by julenuri** [comment]
I like this software under rapps-db.

Bad thing, to not have the x64 version in the PR. To be added in the future.  @scropool ?

File: CoreTemp.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Name = Core Temp
+Version = 1.18.1
+License = EULA
+Description = Core Temp is a compact, no fuss, small footprint, yet powerful program to monitor processor temperature and other vital information.
+Category = 12
+URLSite = https://www.alcpu.com/CoreTemp/
+URLDownload = https://www.alcpu.com/CoreTemp/Core-Temp-setup-v1.18.1.0.exe
+SHA1 = f4329a1afc37f143ba1d39d9670ca4b1acd61c23
+SizeBytes = 1294336
+Icon = coretemp.ico
```
[the remark this answers is not in the document]
Comment (scropool) [reply CoreTemp.txt]: I will take your suggestion seriously

[the remark this answers is not in the document]
Comment (scropool) [reply CoreTemp.txt]: > I would delete this line and change the name of the .ico to be the same of the txt.

what about the icon file? won't it be there?

**Comment by scropool** [comment]
> I like this software under rapps-db.
> 
> Bad thing, to not have the x64 version in the PR. To be added in the future. @scropool ?

reactos only runs 64 bit applications with .msi extension so if coretemp works I can add it

**Comment by scropool** [comment]
> I like this software under rapps-db.
> 
> Bad thing, to not have the x64 version in the PR. To be added in the future. @scropool ?

image
x64 version is now working with portable zip edition

**Pull request merged**